README.md

```diff
@@ -3,22 +3,24 @@
 [![Travis](https://travis-ci.org/JuliaDiff/ChainRules.jl.svg?branch=master)](https://travis-ci.org/JuliaDiff/ChainRules.jl)
 [![Coveralls](https://coveralls.io/repos/github/JuliaDiff/ChainRules.jl/badge.svg?branch=master)](https://coveralls.io/github/JuliaDiff/ChainRules.jl?branch=master)
 
-**Docs:* 
+**Docs:**
 [![](https://img.shields.io/badge/docs-master-blue.svg)](https://JuliaDiff.github.io/ChainRules.jl/dev)
 [![](https://img.shields.io/badge/docs-stable-blue.svg)](https://JuliaDiff.github.io/ChainRules.jl/stable)
 
 The ChainRules package provides a variety of common utilities that can be used by downstream automatic differentiation (AD) tools to define and execute forward-, reverse-, and mixed-mode primitives.
 
-This package is a WIP; the framework is essentially there, but there are a bunch of TODOs, virtually no tests, etc. PRs welcome! Documentation is incoming, which should help if you'd like to contribute.
+The Core logic of ChainRules is implemented in [ChainRulesCore.jl](https://github.com/JuliaDiff/ChainRulesCore.jl).
+If adding ChainRules support to your package (defining `rrule`s or `ffules` you only need to depend on that very light-weight package.
+This repo: ChainRules.jl is what people actually use, it reexports all the ChainRules functionality,
+and has all the rules for the julia standard library.
 
-Here are some of the basic goals for the package:
 
-- First-class support for complex differentiation via Wirtinger derivatives.
+Here are some of the core features of the package:
 
 - Mixed-mode composability without being coupled to a specific AD implementation.
-
+- Extensible rules: package authors can add rules (and thus AD support) to the functions in their packages in their packages. They don't need to edit the ChainRules repo.
+- Control-inverted design: rule authors can fully specify derivatives in a concise manner while naturally allowing the caller to compute only what they need.
```

Comment:
what does this mean?

Reply:
actually i think i know what it means, but it's taken me a couple readings to get there...

How about changing the last bit to something like this (still not totally happy with it)

```suggestion
- Control-inverted design: rule authors can fully specify derivatives in a concise manner that supports computation efficiencies, so we will only compute as much as the user requests.
```